sky/templates/kubernetes-port-forward-proxy-command.sh.j2

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+set -uo pipefail
+
+# Checks if socat is installed
+if [ ! "$(which socat)" ]; then
+  echo "Using 'port-forward' mode to run ssh session on Kubernetes instances requires 'socat' to be installed. Please install 'socat'" >&2
+  exit
+fi
+
+# If the port-forward command is already running on the jump pod, kill it
+pgrep -f "kubectl port-forward svc/{{ ssh_jump_name }} {{ local_port }}:22" > /dev/null
+if [ $? -eq 0 ]; then
+    pkill -f "kubectl port-forward svc/{{ ssh_jump_name }} {{ local_port }}:22"
+fi
+
+# Establishes connection between local port and the ssh jump pod
+kubectl port-forward svc/{{ ssh_jump_name }} {{ local_port }}:22 &
+# Terminate the port-forward process when this script exits.
+K8S_PORT_FWD_PID=$!
+trap "kill -9 $K8S_PORT_FWD_PID" EXIT
+
+# checks if a conection to local_port of 127.0.0.1:[local_port] is established 
+while ! nc -z 127.0.0.1 {{ local_port }}; do
+    sleep 0.1
+done
+
+# Establishes two directional byte streams to handle stdin/stdout between
+# terminal and the jump pod.
+# socat process terminates when port-forward terminates.
+socat - tcp:127.0.0.1:{{ local_port }}
```

Comment:
On running any command (sky launch or ssh), stdout prints:
```
Connection to 127.0.0.1 port 23100 [tcp/*] succeeded!
```
Is there some way to suppress this message?

Reply:
Doesn't seem like there is a clear way to go about this especially since we are utilizing socat's stdout/stderr to interact with the node. Wasn't able to find a clear way from [this man page](https://linux.die.net/man/1/socat) as well. Also, it looks like the message is OS specific? I'm using Debian GNU/Linux 10 (buster) and am not seeing the message:

```
$ ssh sky-1aad-gcpuser
Warning: Permanently added '[127.0.0.1]:23100' (ECDSA) to the list of known hosts.
Warning: Permanently added 'xx.xxx.x.x' (ECDSA) to the list of known hosts.
Linux sky-1aad-gcpuser-ray-head 4.19.0-22-cloud-amd64 #1 SMP Debian 4.19.260-1 (2022-09-29) x86_64

The programs included with the Debian GNU/Linux system are free software;
the exact distribution terms for each program are described in the
individual files in /usr/share/doc/*/copyright.

Debian GNU/Linux comes with ABSOLUTELY NO WARRANTY, to the extent
permitted by applicable law.
Last login: Tue Aug 29 04:19:04 2023 from xx.xxx.x.x
```